index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
+    />
+    <title>Portfolio</title>
+</head>
+  <body>
+    <header class="navbar">
+      <ul>
+        <li class="navbar-logo">Tadesse J.</li>
+        <li class="navbar-toggle"><i class="fa fa-bars"></i></li>
+        <li class="navbar-links"><a href="#"></a></li>
+        <li class="navbar-links"><a href="#"></a></li>
+        <li class="navbar-links"><a href="#"></a></li>
+        
+    </header>
+      <!-- HOME SECTION -->
+      <section class="container-fluid">
+          <div>
+            <h1 class="heading">
+              Hey There.
+              I’m Tadesse
+            </h1>
+            <h2 class="card-title">I’m a software developer</h2>
+            <p class="card-text">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have a
+              project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+        <div>
+          <ul class="element">
+            <li>
+              <a href="#">
+                <i class="bi bi-twitter text-white"></i>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <i class="bi bi-linkedin text-white"></i>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <i class="bi bi-instagram text-white"></i>
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <i class="bi bi-github text-white"></i>
+              </a>
+            </li>
+          </ul>
```

Comment:
instead of changing the color of the social media icons in HTML use CSS correlated with the HTML class name.